### PR TITLE
User login UI

### PR DESCRIPTION
**Pull request by yuanshu777**
Added the Login UI as our launch page for now
Added some colours code to use

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.2.2' apply false
-    id 'com.android.library' version '7.2.2' apply false
+    id 'com.android.application' version '7.3.1' apply false
+    id 'com.android.library' version '7.3.1' apply false
 }
```
Comment (Jacob4649) [inline build.gradle:6]: we probably want to avoid updating the Gradle files

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #Mon Oct 24 10:52:31 EDT 2022
 distributionBase=GRADLE_USER_HOME
```
Comment (Jacob4649) [inline gradle/wrapper/gradle-wrapper.properties:2]: We probably also want to avoid updating this

**Review by Jacob4649 — CHANGES_REQUESTED** [review]
Looks really good, just a few small changes with the gradle files

File: build.gradle (hunk 1 above)
Comment (yuanshu777) [reply build.gradle]: solved

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -13,6 +13,20 @@
...
+        <activity
+            android:name=".frontend.FrontPageActivity"
+            android:exported="false">
+            <meta-data
+                android:name="android.app.lib_name"
+                android:value="" />
+        </activity>
         <activity
             android:name=".frontend.MainActivity"
             android:exported="true">
```
Comment (Enkhzul001) [inline app/src/main/AndroidManifest.xml:32]: What is happening here?

**Review by Jacob4649 — CHANGES_REQUESTED** [review]
Looks good to me, but a more verbose name for `PRegActivity` might be appropriate. Something like `ProviderRegistrationActivity` (provided that's what it is meant to be).

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (yuanshu777) [reply app/src/main/AndroidManifest.xml:32]: I think for each activity, you have to declare them in the AndroidManifest, and the PRehActivity and FronPageActivity are activities that we want to link to the Mainactivity

**Comment by yuanshu777** [comment]
> Looks good to me, but a more verbose name for `PRegActivity` might be appropriate. Something like `ProviderRegistrationActivity` (provided that's what it is meant to be).

I know, I thought it would be too long. But since PR isn't appropriate, I will change it. Thank you

**Review by Jacob4649 — APPROVED** [review]
Looks good!

**Review by BullDF — APPROVED** [review]
Not too familiar with this UI thing, but the graphics look good (seen during tutorial).

**Review by Enkhzul001 — APPROVED** [review]
Overall, looks great! I will use some of the images and colors to match the style on my UI.

**Pull request merged**